File: write_first_4lines.py
#!/Users/lpipes/anaconda2/bin/python
import glob
path = "Exercise_2"
dir_list = glob.glob(path + '/*fastq.gz/')
for file in dir_list:
	spl = file.split("/")
	file_out = open("Exercise_3/" + spl[1],"w")
	file_in = open(file,"r")
	file_out.write(file_in.readline())
	file_out.write(file_in.readline())
	file_out.write(file_in.readline())
	file_out.write(file_in.readline())
	file_out.close()
	file_in.close()

File: write_first_4lines.pl
#!/usr/bin/perl
use strict;
use warnings;

opendir(DIR,"Exercise_2") || die("cannot open dir!");
while(my $file = readdir(DIR)){
	if ( $file !~ /.fastq.gz$/ ){ next; }
	open(FILE,"Exercise_2/" . $file) || die ("cannot open file!");
	open(OUTFILE,">","Exercise_3/" . $file) || die("cannot open file!");
	my $counter=0;
	while(<FILE>){
		if ( $counter == 4 ){ last; }
		my $line = $_;
		chomp($line);
		print OUTFILE "$line\n";
		$counter++;
	}
	close(OUTFILE);
}
closedir(DIR);
